### PR TITLE
Bitbucket client ID and Secret

### PR DESCRIPTION
**Pull request by babenek**
## Description

Please include a summary of the change and which is fixed.

- United #327 and #326 patterns
- Some credentials in 32 symbols are found with both rules
- Requires https://github.com/Samsung/CredData/pull/33
- Common slowdown with the pattern is 33% (PS: last test found only 23% slowdown for Python3.8 🤔)

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [x] UnitTest 
- [x] Test Benchmars after

### CODE REVIEW

File: .github/workflows/benchmark.yml [hunk 1]
```diff
@@ -62,7 +62,8 @@ jobs:
     - name: Checkout CredData
       uses: actions/checkout@v3
       with:
-        repository: Samsung/CredData
+        repository: babenek/CredData
+        ref: bitbucket
```
Comment (babenek) [inline .github/workflows/benchmark.yml:66]:
```suggestion
        repository: Samsung/CredData
```

File: credsweeper/rules/config.yaml [hunk 2]
```diff
@@ -709,6 +709,28 @@
...
+  severity: medium
+  type: pattern
+  values:
+    - (^|[^0-9A-Za-z.$_/+-])(?P<value>([a-zA-Z0-9_-]{32}){1,2})([^0-9A-Za-z.$_/+-]|$)
+  filter_type: WeirdBase64Token
+  min_line_len: 32
+  usage_list:
+    - src
+    - doc
+
```
Comment (csh519) [inline credsweeper/rules/config.yaml:733]: I think those severity of those patterns should be `INFO`.
Could you please let me know why severities are `MEDIUM` and `HIGH`?

File: credsweeper/rules/config.yaml [hunk 3]
```diff
@@ -709,6 +709,28 @@
...
+  values:
+    - (^|[^0-9A-Za-z.$_/+-])(?P<value>[a-zA-Z0-9]{18}([a-zA-Z0-9]{14})?)([^0-9A-Za-z.$_/+-]|$)
+  filter_type: WeirdBase64Token
+  min_line_len: 18
+  usage_list:
+    - src
+    - doc
+
+- name: Bitbucket Client Secret
+  severity: medium
```
Comment (babenek) [inline credsweeper/rules/config.yaml:724]:
```suggestion
  severity: info
  type: pattern
  values:
    - (^|[^0-9A-Za-z.$_/+-])(?P<value>[a-zA-Z0-9]{18}([a-zA-Z0-9]{14})?)([^0-9A-Za-z.$_/+-]|$)
  filter_type: WeirdBase64Token
  min_line_len: 18
  usage_list:
    - src
    - doc

- name: Bitbucket Client Secret
  severity: info
```

File: credsweeper/rules/config.yaml (hunk 2 above)
Comment (babenek) [reply credsweeper/rules/config.yaml]: Mistake after merge.

Comment (csh519) [reply credsweeper/rules/config.yaml]: Thank you for a quick fix!

**Comment by csh519** [comment]
It seems there are some trouble on benchmark action.
Please check.

**Review by csh519 — APPROVED** [review]
Thank you for adding Bitbucket Rules.

LGTM 👍

**Review by Yullia — APPROVED** [review]

**Pull request merged**